ci-operator/step-registry/openshift/e2e/aws/csi/OWNERS

```diff
@@ -0,0 +1,11 @@
+approvers:
+- smarterclayton
+- wking
+- stevekuznetsov
```

Comment:
Can we use a smaller list here?

Reply:
Hmm, this was supposed to be a symlink to `../OWNERS` as in the other PRs, not sure what happened (the contents are the same).

The original tests don't have any ownership information, so I don't know what that list should be.

Reply:
Replaced it with a symlink.